posthog/hogql/test/__snapshots__/test_query.ambr

```diff
@@ -43,7 +43,7 @@
   '''
   -- ClickHouse
   
-  SELECT arrayMap(x -> multiply(x, 2), [1, 2, 3]), 1 
+  SELECT arrayMap(x -> multiply(x, 2), [1, 2, 3]) AS `arrayMap(x -> multiply(x, 2), [1, 2, 3])`, 1 
```

Comment:
This feels kind of dodgy 😬 

Perhaps it's better to run it through some "sanitize" method that converts it to something like `_arrayMap_x_multiply_x_2_1_2_3` ? Seeing large full SQL expressions as identifiers feels hackable somehow

Reply:
Yeah I was thinking this - we use these names as the column names when we return HogQL to the browser via the API, but some sanitization could be nice here

Reply:
@mariusandra I've got changes locally to do this, but you end up with some inconsistency, for example, if I have the following view:

<img width="806" alt="image" src="https://github.com/user-attachments/assets/428bd068-4922-45ac-a0f7-b06487667577" />

and then query `*` on it, you get:

<img width="874" alt="image" src="https://github.com/user-attachments/assets/dd61a48a-f22f-47a0-9643-ec4dbe04df64" />

I'd rather not change the logic for column names being returned to the browser cos DataViz nodes rely on these names to be consistent for matching visualization settings <> columns

I think this is likely a minor tradeoff, should hopefully encourage users to alias columns too (maybe?) - and I'll see if I can change DataViz logic to not rely on column names so heavily